js/stores/settings_store.js

```diff
@@ -4,7 +4,8 @@ var AppDispatcher = require('../dispatcher/app_dispatcher');
 var ActionTypes = require('../constants/constants').ActionTypes;
 
 var _settings = {
-  cmd: ":"
+  cmd: ':',
+  escape: '\\'
```

Comment:
Escape is one backslash but you have to escape it in the code or else it escapes the closing quote.